MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,251 @@
+                alertTitle = "Поздравляем. Лучший результат!"
+            }
+            let recordDateString = dateToString(date: recordDate)
+            averageAccuracy = Double(allCorrectAnswer * 100) / Double(numberOfQuestionsInGame * gameCount)
+            let resultTemp = QuizeResultsViewModel(
+                title: alertTitle,
+                text: """
+                Ваш результат:\(currentCorrectAnswer)/\(numberOfQuestionsInGame)
+                Количество сыграных квизов: \(gameCount)
+                Рекорд: \(recordCorrectAnswer)/\(numberOfQuestionsInGame) \(recordDateString)
+                Средняя точность: \(String(format: "%.02f", averageAccuracy))%
+                """,
+                buttonText: "Начать новую игру"
+            )
+            show(quize: resultTemp)
+        } else {
+            currentQuestionIndex += 1 // увеличиваем индекс текущего вопроса на 1
+            // показать следующий вопрос
+            show(quize: convert(model: questions[currentQuestionIndex]))
+        }
+    }
 
- Картинка: Old
- Настоящий рейтинг: 5,8
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+    private func startGame() {
+        currentCorrectAnswer = 0
+        currentQuestionIndex = 0
+        questions.shuffle()
+        show(quize: convert(model: questions[currentQuestionIndex]))
+    }
 
+    private func show(quize step: QuizeStepViewModel) {
+        // здесь мы заполняем нашу картинку, текст и счётчик данными
+        counterLabel.text = step.questionNumber
+        posterImage.image = step.image
+        questionLabel.text = step.question
+        posterImage.layer.borderWidth = 0
+    }
 
- Картинка: The Ice Age Adventures of Buck Wild
- Настоящий рейтинг: 4,3
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+    private func show(quize result: QuizeResultsViewModel) {
+        // здесь мы показываем результат прохождения квиза
+        // создаём объекты всплывающего окна
+        let alert = UIAlertController(
+            title: result.title, // заголовок всплывающего окна
+            message: result.text, // текст во всплывающем окне
+            preferredStyle: .alert) // preferredStyle может быть .alert или .actionSheet
+
+        // создаём для него кнопки с действиями
+        let action = UIAlertAction(title: result.buttonText, style: .default) { _ in
+            // убираем затемнение фона
+            UIView.animate(withDuration: 0.25) {
+                self.backgroundView.alpha = 0
+            }
+            self.backgroundView.removeFromSuperview()
+            self.startGame()
+        }
+
+        // добавляем в алерт кнопки
+        alert.addAction(action)
+
+        // затемнение фона
+        view.addSubview(backgroundView)
+        UIView.animate(withDuration: 0.25) {
+            self.backgroundView.alpha = UIColor.YPTheme.background.cgColor.alpha
+        }
+        // показываем всплывающее окно
+        self.present(alert, animated: true, completion: nil)
```

Comment:
```suggestion
        present(alert, animated: true, completion: nil)
```

Используйте короткую версию, каждый раз, когда это возможно